control_algorithms/adrc.py

```diff
@@ -0,0 +1,62 @@
+"""
+Active Disturbance Rejection Control (ADRC) is a robust control strategy
+that estimates and compensates for disturbances in real-time without needing
+an explicit mathematical model of the system.
+
+It consists of:
+1. Tracking Differentiator (TD) - Smooths the reference signal
+2. Extended State Observer (ESO) - Estimates system states and disturbances
+3. Nonlinear State Error Feedback (NLSEF) - Generates the control signal
+
+Refer - https://en.wikipedia.org/wiki/Active_disturbance_rejection_control
+"""
+
+
+class ADRC:
+    def __init__(self, beta1: float, beta2: float, beta3: float, setpoint: float = 0.0):
```

Comment:
Using human names instead of mathematical names helps humans understand.
```suggestion
    def __init__(self, error_correction: float, disturbance: float, acceleration: float, target: float = 0.0):
```